controllers/api/openstack/rename.py

```diff
@@ -0,0 +1,30 @@
+import cherrypy
+from datetime import datetime
+from time import mktime
+from getFunctions import getNovaInstance
+from novaclient.exceptions import ClientException, NotFound
+
+class Rename(object):
+    exposed = True
+
+    '''
```

Comment:
If this is the class's docstring, it should be at the top of the class before `exposed = True`.